_maps/shuttles/whiteship_kilo.dmm

```diff
@@ -246,17 +246,13 @@
 	callTime = 250;
 	can_move_docking_ports = 1;
 	dir = 2;
-	dwidth = 7;
-	height = 17;
 	shuttle_id = "whiteship";
 	launch_status = 0;
 	movement_force = list("KNOCKDOWN"=0,"THROW"=0);
 	name = "Mining Shuttle";
 	port_direction = 4;
-	preferred_direction = 8;
-	width = 13;
-	dheight = 7
-	},
+	preferred_direction = 8
+},
```

Comment:
what da hell happened here

Reply:
I have no idea, but fixing whatever demons lie here is outside the scope of this pr